### PR TITLE
Config with zope product

### PR DESCRIPTION
**Pull request by icemac**
Lint the code.
Add support for Python 3.5, 3.7, 3.8 and 3.9.

### CODE REVIEW

File: buildout.cfg [hunk 1]
```diff
@@ -1,5 +1,5 @@
 [buildout]
-extends = https://raw.githubusercontent.com/zopefoundation/Zope/master/versions.cfg
+extends = http://zopefoundation.github.io/Zope/releases/master/versions.cfg
```
Comment (mauritsvanrees) [inline buildout.cfg:2]: This should be https. (There is a redirect from http to https though, so it probably works.)

File: src/five/customerize/browser.py [hunk 2]
```diff
@@ -27,7 +33,7 @@ def getViews(iface, type=IRequest):
     for reg in gsm.registeredAdapters():
         if (len(reg.required) > 0 and
             reg.required[-1] is not None and
-            reg.required[-1].isOrExtends(type)):
+                reg.required[-1].isOrExtends(type)):
```
Comment (mauritsvanrees) [inline src/five/customerize/browser.py:36]: This extra indent seems strange. But I guess the linter did this automatically. Fine with me.

**Review by mauritsvanrees — APPROVED** [review]
LGTM.
Two minor comments inline, but could be ignored.

File: src/five/customerize/browser.py (hunk 2 above)
Comment (icemac) [reply src/five/customerize/browser.py:36]: It is suggested by PEP-8 (https://www.python.org/dev/peps/pep-0008/#indentation) to make the lines of the if-statement and the block inside it distinguishable.

File: buildout.cfg (hunk 1 above)
Comment (icemac) [reply buildout.cfg]: Good catch!

**Comment by icemac** [comment]
Thank you for reviewing this PR. 😃

**Pull request merged**